### PR TITLE
manually cherry picked affinity for tidb and pump

### PR DESCRIPTION
**Pull request by Yisaer**
cherry-picked by https://github.com/pingcap/tidb-operator/pull/1251

### CODE REVIEW

File: charts/tidb-cluster/values.yaml [hunk 1]
```diff
@@ -454,6 +477,32 @@ binlog:
     # affinity for pump pod assignment, default: empty
     # ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
     affinity: {}
+      ## The following setting is recommended to make each pump instance be distributed deployed in each node
```
Comment (weekface) [inline charts/tidb-cluster/values.yaml]:
```suggestion
    ## The following setting is recommended to make each pump instance be distributed deployed in each node
```

**Comment by DanielZhangQD** [comment]
This does not match with docs https://github.com/pingcap/docs-cn/pull/2083

**Pull request merged**